resources/assets/js/videos/videoContainer.vue

```diff
@@ -281,7 +285,15 @@ export default {
             delete annotation.shape;
 
             return VideoAnnotationApi.save({id: this.videoId}, annotation)
-                .then(this.addCreatedAnnotation, handleErrorResponse)
+                .then((res) => {
+                    this.addCreatedAnnotation(res);
+                    if (tmpAnnotation.track) {
+                        this.disableJobTracking = res.body.trackingJobLimitReached;
+                    }
+                }, (res) => {
+                    handleErrorResponse(res);
+                    this.disableJobTracking = res.status === 429;
+                })
```

Comment:
I noticed that the annotation was no longer flashing in the timeline (to indicate that tracking is in progress, cf. [here](https://github.com/biigle/core/blob/f0f2aeb3c193986d47e75f1390daefdf12e4c54e/resources/assets/js/videos/components/annotationClip.vue#L108)). Git bisect lead me to this change. When I revert it, the flashing works. Can you please check why it doesn't work any more with this change?